lib/msal-node/docs/key-vault-managed-identity.md

```diff
@@ -151,6 +95,59 @@ main();
 
 For an implementation, see the code sample: [auth-code-key-vault](../../../samples/msal-node-samples/auth-code-key-vault).
 
+> :information_source: Converting `PKCS12/PFX` to `PEM`
+>
+> In most circumstances, Azure Key Vault can export certificates and private keys in `pem` format (see: [Export stored certificates](https://docs.microsoft.com/azure/key-vault/certificates/how-to-export-certificate?tabs=azure-cli#export-stored-certificates)), if **Content Type** was chosen as `pem` during certificate generation (see: [Create a certificate in Key Vault](https://docs.microsoft.com/azure/key-vault/certificates/tutorial-rotate-certificates#create-a-certificate-in-key-vault)). If for some reason this is not the case, OpenSSL can be used for conversions:
+>
+> ```bash
+> openssl pkcs12 -in certificate.pfx -out certificate.pem
+> ```
+>
+> If the conversion needs to happen programmatically, then you may have to rely on a 3rd party package, as Node.js offers no native method for this. For instance, using a popular TLS implementation like [node-forge](https://www.npmjs.com/package/node-forge), you can do:
```

Comment:
Seems like this doesn't need to be indented?

Reply:
And are there any other libraries besides `node-forge` we can recommend?

Reply:
I believe we did not find any. Any recommendations?

Reply:
@jasonnutter @sameerag I've found a dozen small pfx to pem packages, but none very popular. [pem](https://www.npmjs.com/package/pem) is a good one that I tried and works fine. Haven't tried [google-p12-pem](https://www.npmjs.com/package/google-p12-pem) but looks promising. Shall we mention these?

Reply:
Looks like **google-p12-pem** is using node-forge under the hood. Will give [jose](https://www.npmjs.com/package/jose) a try